stable/sonarqube/README.md

```diff
@@ -39,6 +39,8 @@ The following table lists the configurable parameters of the Sonarqube chart and
 
 | Parameter                                   | Description                               | Default                                    |
 | ------------------------------------------  | ----------------------------------------  | -------------------------------------------|
+| `serviceAccount.create`                     | Creates a Service Account                 | `false`                                    |
+| `serviceAccount.name`                       | The service account name. Uses existing SA if create set to false                 | `false`                                    |
```

Comment:
Looks like your default is empty for serviceAccount.name

Reply:
I've updated the PR. The tests are failing. They pass locally for me. I'm not sure whats causing the test failure at this point.